server/routes/pokeDex.js

```diff
@@ -0,0 +1,45 @@
+const { response } = require("express");
+const express = require("express");
+const router = express.Router();
+
+const axios = require("axios");
+
+router.get("/", async (req, res) => {
```

Comment:
Any idea what the `res` object here is? When should you use it and how might you use it to improve the experience in the client? (Hint: favoriting Pokemon 😬)

Reply:
As the application was aiming for the bare minimum, it would have been nice to add some features that would let the user know if an action is triggered. A res object would return a response that could be a string to carry to the frontend once the item has been successfully deleted from the database.

Reply:
The usage of one particular method within the `res` object is something I consider to be bare minimum; take a look at the networking tab in your browser for a hint

Reply:
You're so close! `res.status` is in the right direction, but you need to add on to (or replace ✨) that method in order to actually accomplish what you're going for. You're for sure on the right path... maybe there's something helpful [in the docs for that method](https://expressjs.com/en/api.html#res.status) 🙊

Reply:
Okay, awesome. I have looked over the link you have sent. As a result, I have transformed all APIs in the backend to include error handling and response objects to be sent to the client. Depending on the response, the client routes now notifies the user of every action whether the request failed or succeeded. I went ahead and did what you have said on letting the user know what happens by using Chakra-UI Toast and made it into a custom hook to avoid code repeatability.

More on it here: https://github.com/maherhalabi/pokedex-picker/commit/a27e053f5e7c278204f217b8b9f603d721317061